.travis.yml

```diff
@@ -23,3 +24,7 @@ script:
   - make build phaul test/test test/phaul test/piggie
   # Run actual test as root as it uses CRIU.
   - sudo make test phaul-test
+  # This builds crit-go
+  - make -C crit-go/magic-gen lint build magicgen test
+  # Run test of crit-go
+  - sudo make test
```

Comment:
This line is not needed, You are already running the test in line 28. Remove the last two lines.